chainer/functions/noise/dropout.py

```diff
@@ -81,4 +82,4 @@ def dropout(x, ratio=.5, **kwargs):
 
     if configuration.config.train:
         return Dropout(ratio)(x)
-    return x
+    return as_variable(x)
```

Comment:
Please use `return chainer.as_variable(x)`.

Reply:
Reverted. Thank you!